examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>You are mixing <code>B.labels</code> and <code>V('predictions')</code> in one line. Be more consistent.</p><p> </p><p>Also, why not use <code>mode='u'</code> for metrics gathering? See <a href="https://github.com/analysiscenter/batchflow/blob/master/batchflow/pipeline.py#L1180" target="_blank">this docstring</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>